packages/global/components/blocks/pmg-events.marko

```diff
@@ -1,4 +1,4 @@
-$ const { i18n } = out.global;
+$ const { i18n, site, config } = out.global;
```

Comment:
Is `site` used anywhere?  I still it was added but doesn't appear to have a use of it also added.